static/elements/form-field-specs.js

```diff
@@ -571,31 +602,31 @@ export const ALL_FIELDS = {
 
   'dt_milestone_desktop_start': {
     help_text: html`
-      First milestone that allows developers to try 
+      First milestone that allows web developers to try 
       this feature on desktop platforms by setting a flag. 
       When flags are enabled by default in preparation for 
-      removal, please use the fields in the ship stage.`,
+      shipping or removal, please use the fields in the ship stage.`,
   },
 
   'dt_milestone_android_start': {
     help_text: html`
-      First milestone that allows developers to try 
+      First milestone that allows web developers to try 
       this feature on desktop platforms by setting a flag. 
       When flags are enabled by default in preparation for 
-      removal, please use the fields in the ship stage.`,
+      shipping or removal, please use the fields in the ship stage.`,
   },
 
   'dt_milestone_ios_start': {
     help_text: html`
-      First milestone that allows developers to try 
+      First milestone that allows web developers to try 
       this feature on desktop platforms by setting a flag. 
       When flags are enabled by default in preparation for 
-      removal, please use the fields in the ship stage.`,
+      shipping or removal, please use the fields in the ship stage.`,
   },
 
   'flag_name': {
     help_text: html`
-      Name of the flag that enables this feature.'`,
+      Name of the flag on chrome://flags that enables this feature.'`,
```

Comment:
I didn't notice it before, but there is an extra apostrophe at the end of this string.

Reply:
Thanks for noticing that.  The migration was rather messy, with many cases I could not automate consistently.